src/Microsoft.Azure.WebJobs.Host/Scale/TargetScalerResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Represents result for target base scale.
+    /// </summary>
+    public class TargetScalerResult
+    {
+        /// <summary>
+        /// Gets or sets the target worker count.
+        /// </summary>
+        public int WorkerCountDifference { get; set; }
```

Comment:
I don't think scalers should be returning this - instead they should just return a TargetWorkerCount.